nca/Utils/OutputConfiguration.py

```diff
@@ -62,6 +64,23 @@ def print_query_output(self, output, supported_output_formats=None):
                     print(f'Command {dot_cmd_string}\n did not create {path}\n', file=sys.stderr)
                 if os.path.isfile(tmp_dot_file):
                     os.remove(tmp_dot_file)
+            elif self.outputFormat == 'html':
```

Comment:
trying to run the following command from cli:
` python -m nca --connectivity -r tests/fw_rules_tests/podlist/ns_list.json -r tests/fw_rules_tests/podlist/pods_list_4.json -r tests/fw_rules_tests/policies/test2-networkpolicy.yaml -o html -f test2.html`

but getting:
```
Failed to create a svg file: test2.html
'NoneType' object is not iterable
Command dot test2.html.nca_tmp.dot -Tsvg -otest2.html.nca_tmp.svg
 did not create test2.html


```